docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1801,6 +1801,22 @@ use the data from the storage tables, even after the grace period expired.
 Dropping a materialized view with {doc}`/sql/drop-materialized-view` removes
 the definition and the storage table.
 
+#### Incremental refresh limitations
+
+The connector supports incremental refreshes for materialized views with the
+following limitations:
+
+- Appends can only occur between `fromSnapshot`, time of last refresh, and
```

Comment:
As it stands here everything is too technical and meaningless to a normal reader .. we need to explain this better .. and we need to also invert it and explain first what is supported .. and then list limitation ..

Things like .. only simple select, only same catalog and schema, and so on ..